nixos/modules/services/misc/n8n.nix

```
@@ -66,7 +66,6 @@ in
         RestrictNamespaces = "yes";
         RestrictRealtime = "yes";
         RestrictSUIDSGID = "yes";
```

```suggestion
        RestrictSUIDSGID = "yes";
        MemoryDenyWriteExecute = "no"; # v8 JIT requires memory segments to be Writable-Executable.
```

Absence of the entry is not the same as the entry being explicitly set to no, along with a comment.

---

Done, thanks, yes - better to be explicit.